lib/src/test/java/com/auth0/android/lock/WebProviderTest.java

```diff
@@ -52,7 +58,42 @@ public void shouldStart() throws Exception {
                 .resume()
                 .get();
 
-        webProvider.start(activity, "my-connection", callback, 123);
+        webProvider.start(activity, "my-connection", map, callback, 123);
+    }
+
+    @Test
+    public void shouldStartWithCustomAuthenticationParameters() throws Exception {
+        Auth0 account = new Auth0("clientId", "domain.auth0.com");
+        account.setOIDCConformant(true);
+        Options options = new Options();
+        options.setAccount(account);
+
+        options.setUseBrowser(true);
+        options.withAudience("https://me.auth0.com/myapi");
+
+        AuthCallback callback = mock(AuthCallback.class);
+        WebProvider webProvider = new WebProvider(options);
+        Activity activity = spy(Robolectric.buildActivity(Activity.class)
```

Comment:
Can't move to setup as some tests change the `options + account` properties. i.e. setting OIDC conformant, or passing the audience. Maybe there's a helper method I can move this logic to..